wsi/physicaldisplay.cpp

```diff
@@ -634,7 +634,8 @@ bool PhysicalDisplay::GetDisplayConfigs(uint32_t *num_configs,
     return false;
   *num_configs = 1;
   if (configs) {
-    configs[0] = 1;
+    configs[0] = FAKE_DISPLAY_CONFIG_ID;
+    config_ = FAKE_DISPLAY_CONFIG_ID;
```

Comment:
Why in a Get function, member variable config_ need to be change?
Seem strange

Reply:
SF only invoke SetActiveConfig when configs.size > 1. 
In no-display case, only 1 fake config is return by hwc.
and then SF will not invoke SetActiveConfig to set this member.

Another way here, we can return 2 configs as 
configs[0] = 0;
configs[1] = 1;
then no need to set the member